pages/Cart/cart.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/pages/Cart/cart.css">
+    <title>Organix</title>
+</head>
+
+
+<body>
+    <nav class="nav fixed bg-light box-shadow">
+        <a class="brand" href="/index.html">Organix </a>
+        <ul class="navbar-nav">
```

Comment:
you can remove this code if it has no use ... as you have written ul navbar-nav class below